examples/medplum-demo-bots/src/twilio/README.md

```diff
@@ -72,6 +72,32 @@ This bot provides a foundation for voice call handling. You can extend it to:
 - Implement interactive voice response (IVR) menus
 - Queue calls or route to available practitioners
 
+## Deploying the Bot
+
+The provided example uses the [twilio SDK](https://www.npmjs.com/package/twilio) to generate TwiML responses, but building bots with this dependency generates files that are too large to deploy to Medplum. To overcome this, the Medplum bot layer comes pre-installed with `twilio`. To prevent your build from bundling the `twilio` code you need to change `esbuild-script.mjs` and set the `twilio` package as an external dependency.
+
+```js
+// esbuild-script.mjs
+import botLayer from '@medplum/bot-layer/package.json' with { type: 'json' };
+
+const botLayerDeps = Object.keys(botLayer.dependencies);
+const additionalExternals = ['twilio'];
+
+esbuild.build({
+  ...
+  external: [...botLayerDeps, ...additionalExternals],
+});
+```
+
+For this to work as expected you also need to be careful with the way you import from `twilio`. Use the following pattern to access internal modules of the `twilio` package.
```

Comment:
Hmm... Interesting and unfortunate.

Do you know why this is the case?

I know that AWS Lambda Layers has some weird subtle differences between CommonJS and ESM.  Is that related?

It would be nice if we could add an explanation why this change is necessary.